gobblin-api/src/main/java/gobblin/password/PasswordManager.java

```diff
@@ -63,12 +67,17 @@ public PasswordManager load(Optional<String> masterPassword) {
             }
           });
 
-  private final Optional<StrongTextEncryptor> encryptor;
+  private Optional<TextEncryptor> encryptor;
 
   private PasswordManager(Optional<String> masterPassword) {
```

Comment:
Please pass the encryptor class as an optional argument in the constructor. Using a static field creates all kinds of race conditions.